Core/src/main/java/com/plotsquared/core/components/ComponentPresetManager.java

```diff
@@ -191,11 +188,13 @@
 
                 BackupManager.backup(player, plot, () -> {
                     plot.addRunning();
+                    QueueCoordinator queue = plot.getArea().getQueue();
```

Comment:
I kinda feel like we should have some utility method to simplify this, as this is also done in the set command, and it'd be nice to have it exposed via API. Dunno how tho

Reply:
Technically QueueProvider#of(com.sk89q.worldedit.world.World) but it's easier to use the PlotArea getQueue than giving every class the WorldUtil and then converting a String to a WorldEdit World

Reply:
I meant the entire component setting, not just the queue getter. Should have been clearer about that 🌵

Reply:
Ahh right, it's exposed in Plot#setComponent(String component, Pattern blocks, @Nullable QueueCoordinator queue) I guess? If the queue isn't null it writes the blocks to the queue and doesn't enqueue. If the queue is null it makes its own and then enqueues. I'll document that everywhere...